### DISCUSSION TITLE
希望可以增加计划任务容器项目定时自动更新任务

### DISCUSSION THREAD

**Discussion by linkiofo** [1Panel-dev/1Panel#5199] [Ideas]
### 1Panel 版本

最新版本

### 请描述您的需求或者改进建议

因为我有些容器项目更新很频繁，希望能增加计划任务容器项目定时自动更新任务

**Comment by wanghe-fit2cloud** [comment]
是不是可以考虑创建 `Shell 脚本` 类型的任务来实现上述需求。
